apps/web/pages/[user]/book.tsx

```diff
@@ -136,22 +166,42 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     booking = await getBooking();
   }
 
+  const isDynamicGroupBooking = users.length > 1 ? true : false;
```

Comment:
Move this to top function, after users declaration maybe?
Also `= users.length > 1` doesn't require ternary as already return true | false